src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -66,30 +55,41 @@ $active-poster-shadows: 0 10px 20px 0 var(--primary--under-50a);
   &__categories {
     display: flex;
     flex-wrap: wrap;
+    margin-top: $spacing--md;
   }
 
   &__authorBox {
     display: flex;
     align-items: center;
-    margin-top: auto;
-  }
-
-  &__avatar {
-    margin-right: auto;
-  }
-
-  &__author {
-    margin-left: $spacing--sm;
     font-size: $font-size--sm;
-    font-weight: $font-weight--500;
-    @include line-clamp-with-overflow(5);
   }
 
-  &__contactEmail {
-    font-size: $font-size--sm;
+  &__thumbnailContainer {
+    flex-grow: 1;
+    // Tiny initial height forces the container to grow to fill remaining space
+    height: 1px;
+    background-size: cover;
+    background-position: center;
+    img {
+      max-width: 100%;
+      max-height: 100%;
+    }
   }
+}
 
-  &__moreInfoUrl {
-    font-size: $font-size--sm;
+.PosterAttendance {
+  margin: 0 0 0 auto;
+  background: opaque-black(0.3);
```

Comment:
I think we preferred using `--content--under` vars recently, specifically it would be `--content--under-30a` in this case

Reply:
Fixed in 1ac9204fdc6deafbf2c36630788246306e6ed0d9